main/res/layout/cache_filter_generic_string.xml

```diff
@@ -54,8 +54,6 @@
     <com.google.android.material.textfield.TextInputLayout
         android:id="@+id/searchtext"
         android:hint="@string/cache_filter_stringfilter_searchtext_hint"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
         style="@style/textinput_dropdown"
```

Comment:
Isn't it necessary to add the new style "textinput_embedded" also to the enclosed "Autocompletetextview" here?

Reply:
Yes, it should - seems I have forgotten it - too many iterations yesterday... Will add it tonight.

Reply:
done